metis-debias/metis-debias-detect-service/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>metis-debias</artifactId>
+    <groupId>eu.europeana.metis</groupId>
+    <version>13-SNAPSHOT</version>
```

Comment:
Also here: perhaps there is no need to have the version start at 13... to be discussed perhaps.